tests/unittests/gradCheckTest.cpp

```diff
@@ -26,6 +26,14 @@
 
 using namespace glow;
 
+class GradCheckBase : public ::testing::TestWithParam<BackendKind> {
+public:
+  ExecutionEngine IP_{GetParam()};
```

Comment:
nit: IP seems to be coming from "interpreter", maybe call this backend_.

Reply:
"IP" stands for Execution Engine in the code.  Will change it to EE_.